<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment>
  * Released as open source by NCC Group Plc - http://www.nccgroup.com/
  *
  * Developed by Gabriel Caudrelier, gabriel dot caudrelier at nccgroup dot com
  *
  * https://github.com/nccgroup/pip3line
  *
  * Released under AGPL see LICENSE for more information
</comment>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pip3line</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Images/decoder.png</normaloff>:/Images/decoder.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuInput">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Analyse"/>
    <addaction name="separator"/>
    <addaction name="actionPip3line_settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp_with_RegExp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_Pip3line"/>
    <addaction name="actionDebug_dialog"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInput"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Tab"/>
   <addaction name="separator"/>
   <addaction name="action_Analyse"/>
   <addaction name="actionMagic"/>
   <addaction name="actionLogs"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionAbout_Pip3line">
   <property name="text">
    <string>About &amp;Pip3line</string>
   </property>
  </action>
  <action name="action_Analyse">
   <property name="icon">
    <iconset resource="gui_res.qrc">
     <normaloff>:/Images/icons/edit-find-10.png</normaloff>:/Images/icons/edit-find-10.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Analyse</string>
   </property>
   <property name="toolTip">
    <string>Run some basic statistical analysis on given input</string>
   </property>
  </action>
  <action name="actionHelp_with_RegExp">
   <property name="text">
    <string>Help with &amp;RegExp</string>
   </property>
  </action>
  <action name="actionPip3line_settings">
   <property name="text">
    <string>Pip3line &amp;settings</string>
   </property>
  </action>
  <action name="actionNew_Tab">
   <property name="icon">
    <iconset resource="gui_res.qrc">
     <normaloff>:/Images/icons/document-new-3.png</normaloff>:/Images/icons/document-new-3.png</iconset>
   </property>
   <property name="text">
    <string>New Tab</string>
   </property>
  </action>
  <action name="actionGet_data_from_URL">
   <property name="text">
    <string>Get data from URL</string>
   </property>
  </action>
  <action name="actionLogs">
   <property name="icon">
    <iconset resource="gui_res.qrc">
     <normaloff>:/Images/icons/system-search-2.png</normaloff>:/Images/icons/system-search-2.png</iconset>
   </property>
   <property name="text">
    <string>Logs</string>
   </property>
   <property name="toolTip">
    <string>Show Logs </string>
   </property>
  </action>
  <action name="actionMagic">
   <property name="icon">
    <iconset resource="gui_res.qrc">
     <normaloff>:/Images/icons/edit-find-and-replace-2.png</normaloff>:/Images/icons/edit-find-and-replace-2.png</iconset>
   </property>
   <property name="text">
    <string>magic</string>
   </property>
   <property name="toolTip">
    <string>Show quick view</string>
   </property>
  </action>
  <action name="actionDebug_dialog">
   <property name="text">
    <string>Debug dialog</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gui_res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Analyse</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAnalyse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>processNewTransformation(TransformWidget*)</slot>
  <slot>on_actionMass_processing_triggered()</slot>
  <slot>onSeparateInputWindow()</slot>
  <slot>onSaveState()</slot>
  <slot>onLoadState()</slot>
  <slot>onReset()</slot>
  <slot>onAnalyse()</slot>
 </slots>
</ui>
